website/docs/unitybuilds.md

```diff
@@ -0,0 +1,29 @@
+Enables Unity Builds in Visual Studio, also known as Jumbo Builds
+
+```lua
+enableUnityBuilds "value"
+```
+
+If no toolset is specified for a configuration, the system or IDE default will be used.
```

Comment:
"toolset"? Seems to be a copy paste...

Reply:
yeah 100%, I wanted to keep the same structure and easiest was to copy paste something, been so busy lately that this ended up being a rushed job.